### PR TITLE
Rafay

### PR DESCRIPTION
**Pull request by MasoodAhmedRafay**
Changes in kitchen view

### CODE REVIEW

File: public/Template/assets/mobirise/css/mbr-additional.css [hunk 1]
```diff
@@ -148,17 +148,17 @@ body {
 }
 .btn-primary,
 .btn-primary:active {
-  background-color: #149dcc !important;
-  border-color: #149dcc !important;
+  background-color: yellowgreen !important;
```
Comment (NevineMGouda) [inline public/Template/assets/mobirise/css/mbr-additional.css:151]: Please return this the way it is. There is a colored theme that is being used across the whole project. And the same for the border-color below.

File: public/Template/assets/mobirise/css/mbr-additional.css [hunk 2]
```diff
@@ -148,17 +148,17 @@ body {
...
   color: #ffffff !important;
 }
 .btn-primary:hover,
 .btn-primary:focus,
 .btn-primary.focus,
 .btn-primary.active {
   color: #ffffff !important;
-  background-color: #0d6786 !important;
-  border-color: #0d6786 !important;
+  background-color: limegreen !important;
```
Comment (NevineMGouda) [inline public/Template/assets/mobirise/css/mbr-additional.css:160]: Please return this the way it is. There is a colored theme that is being used across the whole project. And the same for the border-color below.

File: src/components/Home.vue [hunk 3]
```diff
@@ -52,7 +52,7 @@
 
 <style scoped>
 #headline{
-    color:black;
-    background: linear-gradient(#149dcc, grey, transparent);
+    color: darkslategrey;
+    background: linear-gradient(greenyellow, ghostwhite, transparent);
```
Comment (NevineMGouda) [inline src/components/Home.vue:56]: Please return this the way it is. There is a colored theme that is being used across the whole project. And the same for the border-color below.

**Review by NevineMGouda — CHANGES_REQUESTED** [review]

**Comment by MasoodAhmedRafay** [comment]
Don’t worry, when you commit the two files I sent to you in our repo. I will clone it to my local repo.

________________________________

**Comment by NevineMGouda** [comment]
Merging Rafay's branch pull request

**Pull request merged**